InteractiveHtmlBom/Run.bat

```diff
@@ -1,35 +1,53 @@
 @echo off
+set pathofEDASourceFile=%1
+set FilePath=%~dp0
+
+::delete --show-dialog after frist start up and setting
+set option=--show-dialog
 
+::detect current language of user.    code of Chinese:0804(zh-cn),0404(zh-tw),1004(zh-sg),0C04(zh-hk)
+for /f "delims=" %%a in ('reg query "HKEY_LOCAL_MACHINE\SYSTEM\ControlSet001\Control\Nls\Language" /v InstallLanguage ^| findstr "0804"') do set language=%%a
+set language=%language:~33,37%
+if  "%language%"=="0804" (set language=zh) else if  "%language%"=="0404" (set language=zh) else if  "%language%"=="1004" (set language=zh) else if  /i "%language%"=="0C04" (set language=zh) else (set language=en)
+if %language%==en (
+	call %FilePath%\i18n\en_UK\language_en.bat
+) else (
+	set PYTHONIOENCODING=utf-8
+	chcp 65001
+	call %FilePath%\i18n\zh_CN\language_zh.bat
+)
 echo -------------------------------------------------------------------------------------------------------------------
 echo -------------------------------------------------------------------------------------------------------------------
 echo                                                                                                                                                                                    -
-echo                                  Thank you for using InteractiveHtmlBom
-echo                           https://github.com/openscopeproject/InteractiveHtmlBom
-echo                                         Bat file by Scarrrr0725
+echo                                        %thx4using%                                              
```

Comment:
All of these spaces should be inside the translation string, not here.
Various languages will have various string widths and to center them all amount of spaces should be specific to a translation.